core-helm-chart/templates/deployment.yaml

```diff
@@ -48,6 +48,8 @@ spec:
           value: "{{ .Values.EXTERNAL_HOSTNAME }}"
         - name: EXTERNAL_PORT
           value: "{{ .Values.EXTERNAL_PORT }}"
+        - name: KUBECONFIG_DIR
+          value: "{{ .Values.spaceConfigDirectory }}"
 
       - name: kcp
```

Comment:
Can we add a condition above all of that to include this container only if the space provider for KS has  space provider type == KCP ?